src/pages/kb/data-sources/amazon-redshift.md

```diff
@@ -0,0 +1,38 @@
+---
+category: querying
+parent_category: data-sources
+title: Amazon Redshift
+slug: amazon-redshift
+# toc: true
+---
+
+## How to Configure Security Groups
+
+1. Begin at the Amazon Redshift console navigation pane. Choose **Clusters**.
+2. Open your cluster to the **Configuration** tab.
+3. Under _Cluster Properties_ → _VPC Security Groups_ choose your security
+   group. It will open in the Amazon EC2 console.
+
+![image](/assets/images/docs/gitbook/redshift-vpc-security-groups.png)
+
+4. Choose the **Inbound** tab.
+
+![image](/assets/images/docs/gitbook/redshift-inbound-tab.png)
+
+5. Here you can add a rule that allows traffic into your cluster. You can
+   configure this to accept inbound connections from your Redash host.
+
+{% callout info %}
+
+If you are using hosted Redash click **Edit** and
```

Comment:
This is the same even if you're using a self hosted one. The only change is the IP you would use -- and there I would put a note on what IP to use in case you're using the hosted version.

Reply:
Fixed in 4bf5f8e